FWCore/Framework/src/EventProcessor.cc

```diff
@@ -397,7 +388,24 @@ namespace edm {
     unsigned int nConcurrentLumis =
         optionsPset.getUntrackedParameter<unsigned int>("numberOfConcurrentLuminosityBlocks");
     if (nConcurrentLumis == 0) {
-      nConcurrentLumis = nConcurrentRuns;
+      nConcurrentLumis = 2;
+    }
+    if (nConcurrentLumis > nStreams) {
+      nConcurrentLumis = nStreams;
+    }
+    std::vector<std::string> loopers = parameterSet->getParameter<std::vector<std::string>>("@all_loopers");
+    if (!loopers.empty()) {
+      //For now loopers make us run only 1 transition at a time
+      nStreams = 1;
+      nConcurrentLumis = 1;
+      nConcurrentRuns = 1;
+    }
+    if (nThreads > 1 or nStreams > 1) {
```

Comment:
Should we move this to be within an `else` block from `if(dumpOptions)`? That way we would only print the info once.

Reply:
Done. This seems better. We don't need the LogInfo stuff in the unit test output.

I viewed the dumpOptions as something for Framework unit tests. I am wondering if it could get used for other purposes... I tried to leave the other output unmodified, just in case there is something that uses it and depends on it being in the log file.

(I'll push all the commits with changes when I get them all done.)